python/cuml/linear_model/elastic_net.pyx

```diff
@@ -224,10 +224,10 @@ class ElasticNet(Base, RegressorMixin):
         """
         self._set_n_features_in(X)
 
-        self.cuElasticNet.fit(X, y, convert_dtype=convert_dtype)
+        self.solver_model.fit(X, y, convert_dtype=convert_dtype)
 
-        self.coef_ = self.cuElasticNet.coef_
-        self.intercept_ = self.cuElasticNet.intercept_
+        self.coef_ = self.solver_model.coef_
+        self.intercept_ = self.solver_model.intercept_
```

Comment:
we shouldn't need to assign these anymore, right? Given the logic in the base class, since this attributes are in the base class these would be returned from the solver, no?